File: sdkwork-backend-sdk-go/types/plus_chat_message_content_form.go
package types

// Chat message creation form
type PlusChatMessageContentForm struct {
	Role string `json:"role"`
	Type string `json:"type"`
	Status string `json:"status"`
	ConversationId int `json:"conversationId"`
	ConversationUuid string `json:"conversationUuid"`
	AgentId int `json:"agentId"`
	AgentType string `json:"agentType"`
	AgentBizType string `json:"agentBizType"`
	Content MessageBody `json:"content"`
	Metadata map[string]interface{} `json:"metadata"`
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_chat_message_content_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusChatMessageContentForm:
    """Chat message creation form"""
    role: str
    type: str
    status: str
    conversation_id: int
    conversation_uuid: str
    agent_id: int = None
    agent_type: str
    agent_biz_type: str = None
    content: MessageBody
    metadata: Dict[str, Any]
